systemtest/src/main/java/io/strimzi/systemtest/TestConstants.java

```diff
@@ -403,6 +406,11 @@ public interface TestConstants {
      */
     String QUOTAS_PLUGIN = "quotasplugin";
 
+    /**
+     * Tag for tests that covers Tiered Storage integration
+     */
+    String TIERED_STORAGE = "tiered-storage";
```

Comment:
Should this follow the other groups where the `-` is not used?

Reply:
Fixed